meetings/2025-02-27/minutes.md

```diff
@@ -0,0 +1,61 @@
+# 2025-02-27 :  TSC Minutes
+
+## Agenda
+
+* Welcome
+
+* [Minutes/actions from previous meeting](../2025-02-13/minutes.md)
+* Welcome [@jakemas as voting TSC member](https://github.com/pq-code-package/tsc/issues/139)
+
+* Updates from related communities:
+  * [PQCA](https://github.com/PQCA)
+  * [Open Quantum Safe](https://github.com/open-quantum-safe)
+
+* Review status of sub projects:
+
+  * [mlkem-native](https://github.com/pq-code-package/mldsa-native)
+  * [mldsa-native](https://github.com/pq-code-package/mlkem-native)
+  * [mlsda-native](https://github.com/pq-code-package/mldsa-native)
+  * [mlkem-libjade](https://github.com/pq-code-package/mlkem-libjade)
+  * [mlkem-rust-libcrux](https://github.com/pq-code-package/mlkem-rust-libcrux)
+  * [mlkem-native](https://github.com/pq-code-package/mlkem-c-embedded)
+
+* Discussion (if not covered previously)
+
+  * [Other Open TSC issues](https://github.com/orgs/pq-code-package/projects/4/views/1)
+
+* Any other business
+  * Next meeting on 2025-03-13 @ 1600 UTC
+
+## Minutes
+
+* Nigel noted that Jake Massimo was voted in as a voting member on the TSC.
+
+A low attendance and brief meeting (time change) - only the following points noted in respect of the proposed PQCA One-year anniversary blog:
+
+* Matthias confirmed he will check with Hanno and Jake on what we want to do for mlkem-native and mldsa-native.
+
+* Matthias confirmed he will check with the libjade and libcrux teams if they want to write something.
```

Comment:
I don't recall this. My memory is that I stated that it's probably best to reach out to them directly. 
But let me do this as a comment here: @mbbarbosa, @tfaoliveira, @franziskuskiefer if you want libjade/libcrux to be covered in the blog post you need to speak up https://github.com/PQCA/TAC/issues/65

Reply:
thanks. corrected.